### PR TITLE
update example for dex ad samaccountname (#832)

### PR DESCRIPTION
**Pull request by cclhsu**
update example for dex ad samaccountname (#832)

Signed-off-by: cclhsu <clark.hsu@suse.com>

# Describe your changes
Add dex active directory example using samaccountname

# Related Issues / Projects
Please provide links to Bugzilla and other GitHub projects with your description if they are related to the changes.

Relates to:

### CODE REVIEW

File: adoc/admin-security-ldap.adoc [hunk 1]
```diff
@@ -555,4 +555,93 @@ base64 encoded PEM file can be generated by running:
 cat <ROOT_CA_PEM_FILE> | base64 | awk '{print}' ORS='' && echo
 ----
 
+==== Example 3: Dex Active Directory TLS Connector Configuration using sAMAccountName
+Run `kubectl --namespace=kube-system edit configmap oidc-dex-config` to edit Dex ConfigMap.
```
Comment (jenting) [inline adoc/admin-security-ldap.adoc:559]: Please use `kustomize` instead. Ref to `addons/dex/patches/custom.yaml`.

File: adoc/admin-security-ldap.adoc [hunk 2]
```diff
@@ -555,4 +554,94 @@ base64 encoded PEM file can be generated by running:
 cat <ROOT_CA_PEM_FILE> | base64 | awk '{print}' ORS='' && echo
 ----
 
-Then, refer to <<sec-admin-security-rbac-update>> to apply the dex.yaml and <<sec-admin-security-rbac-apply>> to access through Web or CLI.
+Then, refer to <<sec-admin-security-rbac-update>> to apply the Dex `custom.yaml` and <<sec-admin-security-rbac-apply>> to access through Web or CLI.
+
+==== Example 3: Dex Active Directory TLS Connector Configuration using sAMAccountName (`addons/dex/patches/custom.yaml`)
+Configure Dex ConfigMap to use Active Directory using the following template:
+----
+connectors:
```
Comment (jenting) [inline adoc/admin-security-ldap.adoc:562]: Could you please ref to #918? Otherwise, the `skuba addon upgrade apply` would not work. Thanks.

Comment (cclhsu) [reply adoc/admin-security-ldap.adoc:562]: ok. seems like it was merged yesterday, so I was compared with previous example.  BTW, seems like there are other examples need to change to kustomized also.

Comment (jenting) [reply adoc/admin-security-ldap.adoc:562]: yes, you're right.

**Pull request merged**